yarn.lock

```diff
@@ -3052,6 +3052,17 @@ array.prototype.flatmap@^1.2.4:
     es-abstract "^1.18.0-next.1"
```

Comment:
I wish the diff size on this yarn.lock wasn't so large 😢

Reply:
Be water, my friend. :martial_arts_uniform: